validate/src/embed.rs

```diff
@@ -215,7 +215,13 @@ pub enum EmbedValidationErrorType {
 /// [`TitleTooLarge`]: EmbedValidationErrorType::TitleTooLarge
 /// [`TooManyFields`]: EmbedValidationErrorType::TooManyFields
 pub fn embed(embed: &Embed) -> Result<(), EmbedValidationError> {
-    let mut total = 0;
+    let length = length(embed);
+
+    if length > EMBED_TOTAL_LENGTH {
```

Comment:
```suggestion
    if length(embed) > EMBED_TOTAL_LENGTH {
```

Reply:
the reason i had to assign this to a variable was because length is also used in ```rs
        return Err(EmbedValidationError {
            kind: EmbedValidationErrorType::EmbedTooLarge { chars: length },
        });
```